utils/zerovec/src/ule/mod.rs

```diff
@@ -34,13 +34,17 @@ use core::{fmt, mem, slice};
 /// Safety checklist for `ULE`:
 ///
 /// 1. The type *must not* include any uninitialized or padding bytes.
-/// 2. The impl of [`ULE::validate_byte_slice()`] *must* return an error if the given byte slice
-///    would not represent a valid slice of this type.
+/// 2. The type must have an alignment of 1 byte.
 /// 3. The impl of [`ULE::validate_byte_slice()`] *must* return an error if the given byte slice
+///    would not represent a valid slice of this type.
+/// 4. The impl of [`ULE::validate_byte_slice()`] *must* return an error if the given byte slice
 ///    cannot be used in its entirety (if its length is not a multiple of `size_of::<Self>()`).
-/// 4. All other methods *must* be left with their default impl, or else implemented according to
+/// 5. All other methods *must* be left with their default impl, or else implemented according to
 ///    their respective safety guidelines.
-/// 5. Acknowledge the following note about the equality invariant.
+/// 6. Acknowledge the following note about the equality invariant.
```

Comment:
Nit: The master checklist contains 6 items, but most safety docs list 5. I think item 4 is the one that most of the safety docs are missing. (This is a pre-existing problem.) Technically item 4 is a subset of item 3, but I wanted to list it separately because it's easy to miss.